index.bs

```diff
@@ -324,13 +325,60 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
+    1. The [=public key credential source=], without its [=public key credential source/id=] item, encrypted so only the owning
+        authenticator can decrypt it.
+
+    [=[RPS]=] don't need to distinguish these two kinds of Credential IDs.
+
 : <dfn>Credential Public Key</dfn>
 :: The public key portion of an [=[RP]=]-specific <dfn>credential key pair</dfn>, generated by an [=authenticator=] and
     returned to an [=[RP]=] at [=registration=] time (see also [=public key credential=]). The private key portion of the
     [=credential key pair=] is known as the <dfn>credential private key</dfn>. Note that in the case of [=self
     attestation=], the [=credential key pair=] is also used as the [=attestation key pair=], see [=self attestation=]
     for details.
 
+: <dfn>Public Key Credential Source</dfn>
+:: A [=credential source=] that can generate [=public key credentials=].
+
+    This consists of a [=struct=] with the following [=struct/items=]:
+    <dl dfn-for="public key credential source">
+        : <dfn>type</dfn>
+        :: A {{PublicKeyCredentialType}}, defaulting to {{public-key}}.
+        : <dfn>id</dfn>
+        :: A [=Credential ID=].
+        : <dfn>privateKey</dfn>
+        :: The [=credential private key=].
+        : <dfn>rpId</dfn>
+        :: The [=Relying Party Identifier=] for the [=[RP]=] that created this credential source.
+        : <dfn>userHandle</dfn>
+        :: The [=user handle=] associated when this credential source was created.
+        : <dfn>otherUI</dfn>
+        :: Other information used by the authenticator to inform its UI. For example, this might include the user's
+            {{displayName}}.
+    </dl>
+
+    The [=authenticatorMakeCredential=] operation creates [=public key credential sources=] and returns the matching
+    [=credential public key=] with which a [=[RP]=] can verify the [=public key credentials=] it creates.
+
+
+: <dfn>Public Key Credential</dfn>
+:: An instance of {{PublicKeyCredential}} in which {{PublicKeyCredential/response}} holds an {{AuthenticatorAssertionResponse}}
+    proving possession of a particular [=credential private key=].
+
+    Note: This is a [=willful violation=] of [[RFC4949]]. In English, a "credential" is both a) the thing presented to prove a
+    statement and b) intended to be used multiple times. It's impossible to achieve both criteria with a single piece of data in
+    a public key system. [[RFC4949]] chooses to define a credential as the thing that can be used multiple times (the public
+    key), while this specification chooses to define a credential as the proof.
+
+    A [=public key credential=] is created by the [=authenticatorGetAssertion=] operation using a [=public key credential
+    source=] created by a previous [=authenticatorMakeCredential=] call. The [=[RP]=] verifies it using the [=credential public
+    key=] returned from that [=authenticatorMakeCredential=] call via {{PublicKeyCredential/[[Create]]()}}.
```

Comment:
I think this is confusing: when I call <something>Credential.create I would expect to get a credential back - not a credential source.